docs/topics/mta-6-installing-web-console-on-openshift.adoc

```diff
@@ -75,3 +75,39 @@ You can install the {ProductName} ({ProductShortName}) and the {WebName} on Open
 ** *User name*: admin
 ** *Password*: Passw0rd!
 . When prompted, create a new password.
+
+== Memory testing requirements for {ProductName}
```

Comment:
The section lacks context. These requirements only apply when installing the tool in OpenShift Local, and the way this reads, even on the section title, it feels like this applies on all scenarios. It needs to be more specific.

Reply:
I suggest we change to **Memory testing requirements for installing on OpenShift Local**

Does this work? @rromannissen

Reply:
@anarnold97 sounds great to me!

Reply:
@rromannissen @CarmiWisemon - had another look at this section (Starting line 81 to 106).

Should we be more explicit about the tests to highlight it is MTA Binary Analysis on OpenShift local? 

Trying to focus on the context. Possibly (please, can we have a comma after "The more memory" as it improves readability to me - but it is me being pedantic? ): 

**OpenShift local requires a minimum amount of memory to complete the MTA analysis. The more memory, the quicker the analysis is completed. The following table describes the various amounts of memory required by OpenShift local to complete the MTA binary analysis:

|TABLE|
| ***** |
| Details |

The analysis tests were completed by running the MTA binary analysis through the MTA UI.

The minimum amount of memory required to run MTA binary analysis using OpenShift local is 12 GB.**

Reply:
I updated the topic based on @anarnold97 @rromannissen  comments. Is this OK?